samples/go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	gocloud.dev/pubsub/rabbitpubsub v0.15.0
 	gocloud.dev/runtimevar/etcdvar v0.15.0
 	gocloud.dev/secrets/hashivault v0.15.0
+	gocloud.dev/secrets/vault v0.15.0
```

Comment:
I don't think there should be any `go.mod` or `go.sum` changes in this PR, and I'm not sure where `gocloud.dev/secrets/vault` is coming from.

Reply:
done